.github/workflows/ci.yml

```
@@ -18,8 +18,9 @@ jobs:
       fail-fast: false
       matrix:
         ruby_version:
-          - "2.7"
+          - "3.3"
           - "3.2"
+          - "3.1"
         include:
           - ruby-version: '2.7'
```

I'm surprised this doesn't run. This suggests to me that the whole `include` was broken.

Regardless, this should already have been
```suggestion
          - ruby_version: '2.7'
```